packages/styles/src/typography/typography.scss

```diff
@@ -0,0 +1,33 @@
+@import url("https://fonts.googleapis.com/css2?family=Arimo:wght@400;600&display=swap");
+
+// Set font family and base size at a root level
+body {
+  color: theme("font.color.default");
+  font-family: theme("font.family"), sans-serif;
+  font-size: theme("font.size.default");
+}
+
+$sizes: ("h1", "h2", "h3", "h4", "h5", "default", "sm", "xs");
```

Comment:
Is there a way for this and `$colors` to read from the JSON config instead? I've only tried using the `theme()` function and that doesn't work